assert/clean.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+function list_files() {
+    find . -not -path '.' -not -path './setup-cli*'
+}
+
+# The ./setup-cli path is expected to be present.
+if test -n "$(list_files)" ; then
```

Comment:
```suggestion
if [ "$(list_files)" != "" ]; then
```

Reply:
Changed to double brackets (single is a can of worms with empty strings since it's a subcommand).